server/webapp/WEB-INF/rails.new/app/views/shared/_footer.html.erb

```diff
@@ -1,10 +1,12 @@
 <ul class="copyright" onclick="dashboard_periodical_executer.stop();">
-    <li>Copyright &copy; 2014 <a href="http://www.thoughtworks.com/products" target='_blank'>ThoughtWorks, Inc.</a> Licensed under <%= link_to(l.string('APACHE_LICENSE_2_0'), "http://www.apache.org/licenses/LICENSE-2.0", :target => '_blank') -%>. Go includes <%= link_to(l.string('THIRD_PARTY_SOFTWARE'), url_for_path('NOTICE/cruise_notice_file.pdf'), :target => '_blank') -%>.</li>
+    <li>Copyright &copy; 2015 <a href="http://www.thoughtworks.com/products" target='_blank'>ThoughtWorks, Inc.</a> Licensed under <%= link_to(l.string('APACHE_LICENSE_2_0'), "http://www.apache.org/licenses/LICENSE-2.0", :target => '_blank') -%>. Go includes <%= link_to(l.string('THIRD_PARTY_SOFTWARE'), url_for_path('NOTICE/cruise_notice_file.pdf'), :target => '_blank') -%>.</li>
     <li class="last"><%=l.string("GO_VERSION")-%>: <%=version%></li>
 </ul>
 <ul class="links">
     <li><%= link_to("(cc) " +  l.string("CCTRAY_FEED"), url_for_path('cctray.xml')) -%></li>
+    <li><%= link_to(l.string("APIS"), "http://www.go.cd/documentation/user/current/api/go_api.html", :target => "_blank") -%></li>
+    <li><%= link_to(l.string("PLUGINS"), "http://www.go.cd/community/plugins.html", :target => "_blank") -%></li>
+    <li><%= link_to(l.string("COMMUNITY"), "http://www.go.cd/community/resources.html", :target => "_blank") -%></li>
```

Comment:
Should we have a "Developers" link, instead, going to a new page on go.cd? Or, "Extend Go", as one link, rather than two?